.github/workflows/main.yml

```
@@ -26,15 +26,20 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
+    # This token is used to download all actions and for all subsequent GitHub API calls.
+    token: ${{ secrets.GH_USER_TOKEN }}
```

actionlint (1.7.8)

unexpected key "token" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)